library/build.gradle

```diff
@@ -30,9 +30,9 @@ repositories {
 
 ext {
     brazeVersion = "3.0.0"
-    engineVersion = "[8.4.0,)"
+    engineVersion = "[9.6.0,)"
```

Comment:
Can we have this be the format that would work for 9.6.0 until 9.X.X, but not for 10.0.0 or higher?